sphinx/builders/html/__init__.py

```diff
@@ -263,14 +264,35 @@ def init_templates(self) -> None:
 
     def init_highlighter(self) -> None:
         # determine Pygments style and create the highlighter
+        style = 'sphinx'
         if self.config.pygments_style is not None:
             style = self.config.pygments_style
         elif self.theme:
             style = self.theme.get_config('theme', 'pygments_style', 'none')
-        else:
-            style = 'sphinx'
```

Comment:
It seems this change is not related to the goal. Could you revert this please?

Reply:
Yes, just wanted to make it consistent with my own

Reply:
Yes sure